Tasks/AppCenterDistributeV2/task.json

```diff
@@ -0,0 +1,200 @@
+{
+    "id": "B832BEC5-8C27-4FEF-9FB8-6BEC8524AD8A",
+    "name": "AppCenterDistribute",
+    "friendlyName": "App Center Distribute",
+    "description": "Distribute app builds to testers and users via App Center",
+    "helpUrl": "https://aka.ms/appcentersupport/",
+    "helpMarkDown": "For help with this task, visit the Visual Studio App Center [support site](https://aka.ms/appcentersupport/).",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 0,
```

Comment:
You can optionally make the minor version the sprint you're adding/editing the task ("149"). Makes it a little easier to inspect task output and know when the code was written.